### PR TITLE
fixes #588: Prominent 'Get started' button

### LINKED ISSUE #588: Make "Get Started" button more prominent
## Summary

- [ ] Center "Get Started" button and make it a larger, more prominent call to action

## Motivation

We want users to be well-aware that they should click this to enter and start our app

### PR DESCRIPTION
**Pull request by crispinamuriel**
fixes #588

### Please provide enough information so that others can review your pull request:
Making get started button bigger/more prominent.

### Explain the **details** for making this change. What existing problem does the pull request solve? 
Used makeStyles object to enlarge button with css.

### Test plan (required)

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Screen Shot 2020-04-20 at 3 34 40 PM

### Final Checklist

- [x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [ ] Have you added any new tests necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x] Did you use yarn, not npm?
- [x] Did you use Material-UI wherever possible?
- [x] Did you format according to Prettier?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

**Comment by crispinamuriel** [comment]
I wasn't sure how big to make the button, or if to change the fontweight, etc. A second pair of eyes would be nice on this one! Many thanks!

File: client/src/components/Login.jsx [hunk 1]
```diff
@@ -58,7 +61,7 @@ const Login = props => {
           <Button variant="login" className={classes.Button} onClick={onClick}>
             {t('login.buttonText')}
           </Button>
-          <TranslationsMenu />
+          <TranslationsMenu /> hi
```
Comment (SomeMoosery) [inline client/src/components/Login.jsx:64]: Let's remove this

Comment (crispinamuriel) [reply client/src/components/Login.jsx:64]: Will do!

Comment (crispinamuriel) [reply client/src/components/Login.jsx]: Screen Shot 2020-04-20 at 3 48 58 PM
This is what it looks like full screened (desktop). I'll try to move the button to the center (vertically) like you suggested!

**Comment by crispinamuriel** [comment]
Screen Shot 2020-04-20 at 4 01 27 PM
Screen Shot 2020-04-20 at 3 34 40 PM
This is my last commit, 'vertical'. I' apologize if I am missing the target with your feedback. Let me know if I should change it again!

File: client/src/components/Login.jsx [hunk 2]
```diff
@@ -19,7 +19,11 @@ const useStyles = makeStyles(theme => ({
   },
   Button: {
     ...buttonsCss.buttons,
-    marginTop: '5vh',
+    fontSize: '28px',
+    height: '50px',
+    lineHeight: '10px',
+    marginTop: '8%'
+
```
Comment (AdhamAH) [inline client/src/components/Login.jsx:26]: try adding this here 
```
   animation: `$glowing 1075ms infinite`,
```

File: client/src/components/Login.jsx [hunk 3]
```diff
@@ -19,7 +19,11 @@ const useStyles = makeStyles(theme => ({
...
   },
   logo: {
```
Comment (AdhamAH) [inline client/src/components/Login.jsx:28]: then add before logo
```

  '@keyframes glowing': {
    '0% ': { backgroundColor: '#2ba805', boxShadow: '0px 1px 10px 0px #4760ff' },
    '50%': { backgroundColor: '#49e819', boxShadow: '0px 1px 13px 3px #4760ff' },
    '100%': { backgroundColor: '#2ba805', boxShadow: '0px 1px 10px 0px #4760ff' },
  },
```

**Comment by acthelemann** [comment]
Looks pretty good. I'd try what @AdhamAH is suggesting as well as using the `Typography` component inside the button instead of using `font-size` in the styles. Maybe `variant="h6"`?

**Comment by AdhamAH** [comment]
By the way @crispinamuriel you can get creative if you want It is just a suggestion and an example.
Otherwise looks good!

**Comment by crispinamuriel** [comment]
Okay it's faster than this gif but the button glows now!!! Thanks @AdhamAH.

Thanks all for the help!

**Comment by SomeMoosery** [comment]
The one thing I'll note is if using `Typography` inside of a `Button` - make sure that the button is clickable everywhere, and not just on the text itself as I've had this issue pop up for me before

**Comment by crispinamuriel** [comment]
I didn't end up using typography it's clickable everywhere!

**Pull request merged**